sa/sa.go

```diff
@@ -436,11 +444,11 @@ func (ssa *SQLStorageAuthority) GetCertificateStatus(ctx context.Context, serial
 
 // NewRegistration stores a new Registration
 func (ssa *SQLStorageAuthority) NewRegistration(ctx context.Context, reg core.Registration) (core.Registration, error) {
+	reg.CreatedAt = ssa.clk.Now()
```

Comment:
Since `registrationToModel` internally copies the `CreatedAt` field and now returns an interface it makes sense to just to do it before passing it into `registrationToModel` so that we don't have to do a gross type cast later.